package.json

```diff
@@ -32,19 +32,21 @@
     "test": "turbo test",
     "test:rust": "turbo test:rust"
   },
+  "pnpm": {
+    "overrides": {
+      "typescript": "5.5.3"
+    }
+  },
   "dependencies": {
     "@buf/connectrpc_eliza.bufbuild_es": "1.10.0-20230913231627-233fca715f49.1",
     "@buf/connectrpc_eliza.connectrpc_es": "1.4.0-20230913231627-233fca715f49.3",
-    "@buf/cosmos_ibc.bufbuild_es": "1.10.0-20240703151202-d7191877c701.1",
-    "@buf/cosmos_ibc.connectrpc_es": "1.4.0-20240703151202-d7191877c701.3",
-    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.10.0-20240703080008-312294d02bf9.1",
-    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.4.0-20240703080008-312294d02bf9.3",
     "@buf/tendermint_tendermint.bufbuild_es": "1.10.0-20231117195010-33ed361a9051.1",
     "@bufbuild/protobuf": "^1.10.0",
     "@connectrpc/connect": "^1.4.0",
     "@connectrpc/connect-web": "^1.4.0"
   },
   "devDependencies": {
+    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.10.0-20240703080008-312294d02bf9.1",
```

Comment:
this one @buf dep is still necessary because the registry incorrectly fails to declare a dependency